domain/robstoll/atrium-domain-robstoll-jvm/src/main/resources/META-INF/services/ch.tutteli.atrium.domain.kotlin_1_3.creating.ResultAssertions

```diff
@@ -0,0 +1 @@
+ch.tutteli.atrium.domain.robstoll.kotlin_1_3.creating.ResultAssertionsImpl
```

Comment:
This file does not belong here but should be in domain-robstoll-kotlin_1_3-jmv
Note we also need to create a registerService task for JS -> see build.gradle in the root of the project -> `createRegisterJsServicesTask` -> add `createRegisterJsServicesTask('domain-robstoll-kotlin_1_3-js',...)` and run `./gr generateJsRegisterServices` afterwards, should create a `registerServices.kt` in domain-robstoll-kotlin_1_3-js

Reply:
I have done exactly the same, but it is still displaying an error : 

```
* What went wrong:
A problem was found with the configuration of task ':atrium-domain-robstoll-kotlin_1_3-js:generateJsRegisterServices'.
> File 'D:\SRH\shardulsonar-atrium\domain\robstoll\extensions\kotlin_1_3\atrium-domain-robstoll-kotlin_1_3-js\src\main\kotlin\ch\tutteli\atrium\domain\kotlin_1_3\registerServices.kt' s
pecified for property '$2' does not exist.
```